platforms/php/versionsToBuild.txt

```diff
@@ -11,7 +11,10 @@
 7.4.15, 9b859c65f0cf7b3eff9d4a28cfab719fb3d36a1db3c20d874a79b5ec44d43cb8, 42670A7FE4D0441C8E4632349E4FDC074A4EF02D 5A52880781F755608BF815FC910DEB46F53EA312,
 7.4.16, 1c16cefaf88ded4c92eed6a8a41eb682bb2ef42429deb55f1c4ba159053fb98b, 42670A7FE4D0441C8E4632349E4FDC074A4EF02D 5A52880781F755608BF815FC910DEB46F53EA312,
 7.4.24, ff7658ee2f6d8af05b48c21146af5f502e121def4e76e862df5ec9fa06e98734, 42670A7FE4D0441C8E4632349E4FDC074A4EF02D 5A52880781F755608BF815FC910DEB46F53EA312,
+7.4.26, e305b3aafdc85fa73a81c53d3ce30578bc94d1633ec376add193a1e85e0f0ef8, 42670A7FE4D0441C8E4632349E4FDC074A4EF02D 5A52880781F755608BF815FC910DEB46F53EA312,
 8.0.1, 208b3330af881b44a6a8c6858d569c72db78dab97810332978cc65206b0ec2dc, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
 8.0.2, 84dd6e36f48c3a71ff5dceba375c1f6b34b71d4fa9e06b720780127176468ccc, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
 8.0.3, c9816aa9745a9695672951eaff3a35ca5eddcb9cacf87a4f04b9fb1169010251, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
-8.0.11, e3e5f764ae57b31eb65244a45512f0b22d7bef05f2052b23989c053901552e16, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
\ No newline at end of file
+8.0.11, e3e5f764ae57b31eb65244a45512f0b22d7bef05f2052b23989c053901552e16, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
+8.0.13, cd976805ec2e9198417651027dfe16854ba2c2c388151ab9d4d268513d52ed52, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
```

Comment:
Hi @qianz2 it looks like some files are missing the new versions. We could do the following:
1. Add new versions to https://github.com/microsoft/Oryx/blob/main/build/constants.yaml#L117 
2. run `build/generateConstants.sh` which will automatically insert the new version in the corresponding files